lib/jekyll/renderer.rb

```diff
@@ -155,7 +157,7 @@ def place_in_layouts(content, payload, info)
           site.in_source_dir(layout.path)
         ) if document.write?
 
-        if layout = site.layouts[layout.data["layout"]]
+        if layout = site.layouts[layout.data["layout"]] ? site.layouts[layout.data["layout"]].first : nil
```

Comment:
Having trouble reading this line.
Maybe pull it out before the `if` block?

(but you don't need me to be able to read it - you need the person who is clicking merge to be able to read it)